BeatsIOSTests/BeatsIOSTests.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  BeatsIOSTests.swift
+//  BeatsIOSTests
+//
+//  Created by Aloisio Formento Junior on 17/03/22.
+//
+
+import XCTest
+@testable import BeatsIOS
+
+class BeatsIOSTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
```

Comment:
Se não for fazer testes, melhor tirar esse target para não poluir o PR

Reply:
Feito